source/adminguide/host_and_storage_tags.rst

```diff
@@ -31,30 +31,39 @@ There are two types of host tags:
 To explain the behavior of host tags, some examples will be demonstrated with two hosts (Host1 and Host2):
 
 #. Tag setup:
+
     * Host1: h1
     * Host2: h2
     * Offering: h1
+
     When a VM is created with the offering, the deployment will be carried out on Host1, as it is the one that has the tag compatible with the offering.
 
 #. Tag setup:
+
     * Host1: h1
     * Host2: h2,h3
     * Offering: h3
+
     Hosts and offerings accept a list of tags, with comma (,) being their separator. So in this example, Host2 has the h2 and h3 tags. When a VM is created with the offering, the deployment will be carried out on Host2, as it is the one that has the tag compatible with the offering.
 
 #. Tag setup:
+
     * Host1: h1
     * Host2: h2,h3
     * Offering: (no tag)
+
     When the offering does not have tags, it will be possible to deploy the VM on any host.
 
 #. Tag setup:
+
     * Host1: (no tag)
     * Host2: h2
     * Offering: h3
+
     None of the hosts have compatible tags and it will not be possible to deploy a VM with the offering. However, CloudStack ignores this behavior when a host is manually selected.
```

Comment:
```suggestion
None of the hosts have compatible tags and it will not be possible to deploy a VM with the offering. However, CloudStack ignores this behavior when a host is manually selected.
```

Reply:
As above, it might be better to keep the indentation.